client/i18n/locales/english/intro.json

```
@@ -295,11 +295,17 @@
   },
   "relational-databases": {
     "title": "Relational Databases",
-    "intro": ["placeholder"],
+    "intro": [
+      "For these courses, you will use real developer tools and software including VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```

Something about "start you off" doesn't sound right here. Maybe you can change it to:
```suggestion
          "In these courses you start off with the basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```

or

```suggestion
          "These courses start off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```